ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/processor/BlockReplay.java

```diff
@@ -146,7 +146,7 @@ public BlockReplay(
       return Optional.empty();
     }
     final MutableWorldState mutableWorldState =
-        worldStateArchive.getMutable(previous.getStateRoot(), previous.getHash()).orElse(null);
+        worldStateArchive.getWorldState(previous.getStateRoot(), previous.getHash()).orElse(null);
```

Comment:
Looks like there is a lingering type/name issue.  If the return type is a MutableWorldState then the method name should be getMutable or getMutableWorldState.  This should either go to just a `WorldState` or the method name should be `getMutable`

Reply:
Done renamed to getMutable